vaadin-notification-flow-parent/vaadin-notification-flow/src/main/resources/META-INF/resources/frontend/notificationConnector.js

```diff
@@ -0,0 +1,35 @@
+(function () {
+    function copyClassName(notification) {
+        const card = notification._card;
+        if (card) {
+            card.className = notification.className;
+        }
+    }
+
+    const observer = new MutationObserver((records) => {
+        records.forEach((mutation) => {
+            if (mutation.type === 'attributes' && mutation.attributeName === 'class') {
+                copyClassName(mutation.target);
+            }
+        });
+    });
```

Comment:
I would suggest to limit `MutationObserver` to watching for the `class` attribute changes only:

```js
new MutationObserver(() => {
  ...
}, { attributeFilter: ['class'] });
```

Reply:
Oh, I missed it. Nevermind.